projects/1.20-fabric/assets/adaptive-tooltips/adaptivetooltips/lang/zh_cn.json

```
@@ -0,0 +1,64 @@
+{
+	"yacl3.config.adaptivetooltips:config.category.adaptivetooltips": "Adaptive Tooltips",
+	"yacl3.config.adaptivetooltips:config.category.adaptivetooltips.group.content_manipulation": "内容修改",
+	"yacl3.config.adaptivetooltips:config.category.adaptivetooltips.group.content_manipulation.desc": "这组选项修改工具提示的内容",
+	"yacl3.config.adaptivetooltips:config.wrapText": "文本换行",
+	"yacl3.config.adaptivetooltips:config.wrapText.desc": "Adaptive Tooltips 该如何给文本换行",
+	"yacl3.config.adaptivetooltips:config.overwriteVanillaWrapping": "覆盖原版换行",
+	"yacl3.config.adaptivetooltips:config.overwriteVanillaWrapping.desc": "不让原版在GUI的工具提示中应用自己的文本换行，而是使用 Adaptive Tooltip 的",
+	"yacl3.config.adaptivetooltips:config.category.adaptivetooltips.group.positioning": "配置",
+	"yacl3.config.adaptivetooltips:config.category.adaptivetooltips.group.positioning.desc": "这组选项决定了工具提示在屏幕上的显示位置",
+	"yacl3.config.adaptivetooltips:config.prioritizeTooltipTop": "优先显示工具提示顶部",
+	"yacl3.config.adaptivetooltips:config.prioritizeTooltipTop.desc": "当工具提示高度大于屏幕高度时，默认情况下，minecraft会优先调整屏幕底部，裁掉顶部。此选项会将优先级更改为顶部，并裁掉底部。",
```

```suggestion
	"yacl3.config.adaptivetooltips:config.prioritizeTooltipTop.desc": "当工具提示高度大于屏幕高度时，默认情况下，Minecraft 会优先调整屏幕底部，裁掉顶部。此选项会将优先级更改为顶部，并裁掉底部。",
```
大小写统一。
（此外，我记得高版本中英文间不加空格了？）